wire-runtime/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/DeprecatedProto.kt.java.interop

```diff
@@ -8,13 +8,14 @@ import com.squareup.wire.ProtoAdapter
 import com.squareup.wire.ProtoReader
 import com.squareup.wire.ProtoWriter
 import com.squareup.wire.TagHandler
+import com.squareup.wire.WireField
 import kotlin.Deprecated
 import kotlin.Int
 import kotlin.String
 import kotlin.jvm.JvmField
 import okio.ByteString
 
-data class DeprecatedProto(val foo: String? = null, val unknownFields: ByteString = ByteString.EMPTY) : Message<DeprecatedProto, DeprecatedProto.Builder>(ADAPTER, unknownFields) {
+data class DeprecatedProto(@WireField(tag = 1, adapter = "com.squareup.wire.ProtoAdapter.STRING") val foo: String? = null, val unknownFields: ByteString = ByteString.EMPTY) : Message<DeprecatedProto, DeprecatedProto.Builder>(ADAPTER, unknownFields) {
```

Comment:
Not really happy with how this looks, should we put those behind a compiler flag maybe? 
@swankjesse Jeff mentioned the absence of the annotations breaks Misk, is there any functionality that relies on `WireField`?

Reply:
Moved it behind the java interop flag.